presto-prometheus/pom.xml

```diff
@@ -229,4 +229,4 @@
         </profile>
     </profiles>
         
-</project>
\ No newline at end of file
+</project>
```

Comment:
accidental change?

Reply:
Vim adds newlines to the end of files, so sort of accidental. I think text files normally end with newlines right? There's probably a standard

Reply:
Yes, the change is correct